convenience/generate-component/index.mts

```diff
@@ -0,0 +1,118 @@
+import { join, relative } from 'path';
```

Comment:
The script has been adapted from the old one, changing it to .mts and adapting imports, etc.